### PR TITLE
Updating Docker Settings

### PR DESCRIPTION
**Pull request by joshfinnie**
This branch gets the docker container auto-building and hosting the blog again... Not sure if all these changes are needed, but it works now!

### CODE REVIEW

**Comment by joshfinnie** [comment]
@TrackMaven/engineers thoughts?

File: develop_server.sh [hunk 1]
```diff
@@ -1,103 +0,0 @@
-#!/usr/bin/env bash
```
Comment (Nearhan) [inline develop_server.sh]: Why did we remove the server?

Comment (joshfinnie) [inline develop_server.sh]: we don't use it... the application is run through `gulp-connect` This is just an erroneous file.

**Comment by Nearhan** [comment]
@joshfinnie looks good to me!

**Pull request merged**